tutorials/dataframe/df106_HiggsToFourLeptons.C

```diff
@@ -43,10 +44,16 @@ using ROOT::RVecF;
 using ROOT::RDF::RSampleInfo;
 using namespace ROOT::RDF::Experimental;
 
+#define STRINGIFY_(x) #x
+#define STRINGIFY(x) STRINGIFY_(x)
```

Comment:
I'm not sure if we already have machinery for this in ROOT, if so please let me know so I can use that instead of redefining mine.

Reply:
We have one of the two in RConfig:
https://github.com/root-project/root/blob/master/core/foundation/inc/ROOT/RConfig.hxx#L438

However, the other one is not part of central ROOT. I also need to re-implement it:
https://github.com/root-project/root/blob/master/roofit/batchcompute/src/RooBatchCompute.cxx#L95

You think it's worth to have it in RConfig too? What would be your suggestion for the name?